src/common/translation/i18n/en.json

```diff
@@ -41,6 +41,9 @@
       }
     }
   },
+  "notEligible": {
+    "notEligibleText": "Your child is unfortunately not eligible for the Godchildren of Culture program due to their birthdate or home town."
```

Comment:
duplicated `notEligible`
should be `notEligible: { text: 'foo' }`
and notEligible should be inside registration object